gradle/dependencies.gradle

```diff
@@ -78,16 +82,24 @@ versions += [
   mavenArtifact: "3.6.0",
   metrics: "2.2.0",
   mockito: "2.23.4",
+  owaspPlugin: "4.0.2",
   powermock: "2.0.0",
   reflections: "0.9.11",
   rocksDB: "5.15.10",
+  scalafmt: "1.5.1",
   scalatest: "3.0.5",
   scoverage: "1.3.1",
+  scoveragePlugin: "2.5.0",
+  shadowPlugin: "4.0.4",
   slf4j: "1.7.25",
   snappy: "1.1.7.2",
+  spotbugs: "3.1.12",
+  spotbugsPlugin: "1.6.9",
+  spotlessPlugin: "3.23.0",
   zkclient: "0.11",
   zookeeper: "3.4.13",
-  zstd: "1.3.8-1"
+  zstd: "1.3.8-1",
+  versionsPlugin: "0.21.0"
```

Comment:
I think we should call this `gradleVersionsPlugin`.

Reply:
Done, name changed.